Kickstarter-iOS/Views/Cells/SettingsFollowCell.swift

```diff
@@ -0,0 +1,60 @@
+import KsApi
+import Library
+import Prelude
+import Prelude_UIKit
+import ReactiveSwift
+import UIKit
+
+public protocol SettingsFollowCellDelegate: class {
+  /// Called when follow switch is tapped
+  func notifyDelegateShowFollowPrivacyPrompt()
```

Comment:
This could be:

`func settingsFollowCellToggledFollowSwitch(_ cell: SettingsFollowCell)`

Not sure if it's toggled or tapped as we don't pass a state to the method?

Reply:
When the user toggles the switch we want to present a prompt asking if they are certain they want to turn the switch off. From there they can decide whether to go through with toggling the switch off/on. Should I rename it to:
`func settingsFollowCellToggledSwitchPresentedPrompt(_ cell: SettingsFollowCell)` ? 👀

Reply:
Maybe similar to the other one reads better?

```swift
func settingsFollowCellDidPresentPrompt(_ cell: SettingsFollowCell)
```

Reply:
Unless we need to be specific about _which_ prompt 🤔

Reply:
cool! and we don't need to be specific about the prompt